src/style.css

```diff
@@ -0,0 +1,20 @@
+*{
+    font-family: SFMono-Medium,"SF Mono","Segoe UI Mono","Roboto Mono","Ubuntu Mono",Menlo,Consolas,Courier,monospace;
+
+    color: #494949;
+
+    background-color: #EAE7DC;
+    background-color: #D8C3A5;
+    background-color: #8E8D8A;
+    background-color: #E98074;
+    background-color: #E85A4F;
```

Comment:
Why need multiple background-color?

Reply:
This file is to make sure css code in styled-components were written correctly. 
I'll add it into .gitignore to prevent confusion.